### PR TITLE
chore(ci): updating pde2e images versions

### PR DESCRIPTION
**Pull request by ScrewTSW**
### What does this PR do?
* Updates to new version of pde2e images

### Screenshot / video of UI
N/A

### What issues does this PR fix or reference?
#1858

### CODE REVIEW

**Review by odockal — APPROVED** [review]

**Comment by odockal** [comment]
@ScrewTSW You did not test the change in that runs: https://github.com/containers/podman-desktop-extension-ai-lab/actions/runs/11238924263/job/31244780551#step:8:50.

File: .github/workflows/ai-lab-e2e-nightly-windows.yaml [hunk 1]
```diff
@@ -186,7 +186,7 @@ jobs:
           -e OUTPUT_FOLDER=/data \
           -e DEBUG=true \
           -v $PWD:/data:z \
-          quay.io/odockal/pde2e-podman:v0.0.1-windows  \
+          quay.io/odockal/pde2e-podman:v0.0.2-windows  \
             pd-e2e/podman.ps1 \
               -downloadUrl ${{ env.PODMAN_URL }} \
               -version ${{ env.PODMAN_VERSION }} \
```
Comment (odockal) [inline .github/workflows/ai-lab-e2e-nightly-windows.yaml:192]: Switching to 0.0.2 might also mean that you need to remove the version parameter.

Comment (odockal) [reply .github/workflows/ai-lab-e2e-nightly-windows.yaml:192]: Which wouild mean also removing the podman version paramters from the whole file - inputs and default env. vars.

**Review by odockal — CHANGES_REQUESTED** [review]
You can add image versions to be parametrized like here: https://github.com/containers/podman-desktop-e2e/blob/main/.github/workflows/desktop-e2e-test-job-windows.yaml#L49. So it can be changed when run on demand.

Also, now you can use setup.exe instead of zip for podman remote url.

**Review by jeffmaury — APPROVED** [review]
LGTM

File: .github/workflows/ai-lab-e2e-nightly-windows.yaml (hunk 1 above)
Comment (ScrewTSW) [reply .github/workflows/ai-lab-e2e-nightly-windows.yaml]: I believe it's fixed now

**Review by odockal — APPROVED** [review]

**Pull request merged**